packages/pilot/src/pages/Errors/GenericError/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withRouter } from 'react-router-dom'
+import { translate } from 'react-i18next'
+import { compose } from 'ramda'
+import { connect } from 'react-redux'
+import {
+  Button,
+  Modal,
+  ModalContent,
+} from 'former-kit'
+
+import { Message, MessageActions } from '../../../components/Message'
+import Icon from './Icon.svg'
+import { receiveLogout } from '../../Account/actions'
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
aqui como não alteramos nada da função, o `mapDispatchToProps` pode ser passado simplificado como objeto
```suggestion
const mapDispatchToProps = {
  logout: receiveLogout,
}
```

Reply:
deixando com a notação de `object` as ações deixaram de ser disparadas, voltei para a forma como estava mas podemos investigar mais disso :thinking: